### ISSUE TITLE
Does not work with LightWorld

### ISSUE THREAD

**Issue by punk0x29a** [adekto/maid64#3]
hi there!

It does not scale Gamera image :( 

```

 18     maid64.start()
 19     cam:draw(function(l,t,w,h)
-- stuff
 29     end)
 30     maid64.finish()

```

When, for example, Gamera is set to display 640x480 on fullscreen, I just get tiny rectangle in the upper left corner of the screen instead of properly scaled graphics... 
Thanks in advance for help! 

Edit: LightWorld is the problem after all, look below. 
Forum thread with better explanation of my issue: https://love2d.org/forums/viewtopic.php?f=4&t=83124

**Comment by adekto** [comment]
tested Gamera

```
local maid64 = require "maid64"
local gamera = require 'gamera'

function love.load()
    love.window.setMode(640, 640, {resizable=true, vsync=false, minwidth=200, minheight=200, fullscreen =true})
    maid64.setup(64)
    cam = gamera.new(0,0,2000,2000)
    cam:setWorld(0,0,2000,2000)
    rotate = 0   
end
function love.update(dt)
    rotate = rotate + dt*2
end
function love.draw()
    maid64.start()
    cam:draw(function(l,t,w,h)
        love.graphics.circle("fill", rotate,  rotate, 10)
    end)
    maid64.finish()
end

```

this seems to work, but im not setting the window for Gamera, the size that its suposed to be is 64x64 or watever ingame resolution you defined for maid64 but gamera seems to have iseus with that.

not sure why you wanna use the library if you want a simple camera

```
love.graphics.push()
    love.graphics.translate(10, 10)
    --draw
love.graphics.pop()
```

**Comment by punk0x29a** [comment]
Hi there!
Thanks for quick and thorough reply. 

It turns out that the LightWorld is an issue after all - I've moved maid64 start and finish functions to internal draw function of LightWorld and it scales, but results are bad, in example - no shadows at all and maximum brightness applied on all spirtes. 

I'll change the title of this issue.
I've also started forum thread: https://love2d.org/forums/viewtopic.php?f=4&t=83124
